build.py

```diff
@@ -68,8 +68,8 @@
 #
 TRITON_VERSION_MAP = {
     '2.34.0dev': (
-        '23.04',  # triton container
-        '23.04dev',  # upstream container
+        '23.04dev',  # triton container
```

Comment:
@nnshah1  @mc-nv By the way, shouldn't `triton container` version be 23.05dev?

Reply:
Should be , looks like a typo... 
Do you won't me to submit a PR for it?

Reply:
https://github.com/triton-inference-server/server/pull/5714